File: app/src/main/java/xyz/b515/schedule/ScheduleApplication.java
package xyz.b515.schedule;

import android.app.Application;
import android.preference.PreferenceManager;

import xyz.b515.schedule.util.ThemeHelper;

public class ScheduleApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        ThemeHelper.getInstance().init(this, PreferenceManager.getDefaultSharedPreferences(this).getInt("theme", 0));
    }

}

File: app/src/main/kotlin/xyz/b515/schedule/ScheduleApplication.kt
package xyz.b515.schedule

import android.app.Application
import android.preference.PreferenceManager

import xyz.b515.schedule.util.ThemeHelper

class ScheduleApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        ThemeHelper.init(this, PreferenceManager.getDefaultSharedPreferences(this).getInt("theme", 0))
    }

}
